docs/cli-reference.adoc

```diff
@@ -1,4 +1,4 @@
-= Overview of the odo (OpenShift Do) CLI Structure
+= Overview of the Odo (OpenShift Do) CLI Structure
```

Comment:
The typography standard that we agreed on is that odo will be written with lowercase `o` :-)

Reply:
Oh, this comes from the template, right? Then it should be updated.

Reply:
yes

Reply:
Added back